planner/core/pb_to_plan.go

```diff
@@ -81,6 +78,17 @@ func (b *PBPlanBuilder) pbToPhysicalPlan(e *tipb.Executor) (p PhysicalPlan, err
 		// TODO: Support other types.
 		err = errors.Errorf("this exec type %v doesn't support yet", e.GetTp())
 	}
+	if subPlan != nil {
+		p.SetChildren(subPlan)
+	}
+	// The limit missed its output cols via the protobuf.
+	// We need to add it back and do a ResolveIndicies for the later inline projection.
```

Comment:
limit schema will not always like what its children are, like its children are projection, and it may have been eliminated